bridges/RadioMelodieBridge.php

```diff
@@ -0,0 +1,34 @@
+<?php
+class RadioMelodieBridge extends BridgeAbstract {
+	const NAME = 'Radio Melodie Actu';
+	const URI = 'https://www.radiomelodie.com/';
+	const DESCRIPTION = 'Retourne les actualités publiées par Radio Melodie';
+	const MAINTAINER = 'sysadminstory';
+	const PARAMETERS = array();
+	const CACHE_TIMEOUT = 3600;
```

Comment:
The last two parameters are default values and can be removed, see [Wiki](https://github.com/RSS-Bridge/rss-bridge/wiki/BridgeAbstract#basic-metadata). I just add the default value for CACHE_TIMEOUT to the Wiki.